src/dataset-operator/pkg/apis/com/v1alpha1/dataset_types.go

```diff
@@ -16,6 +16,7 @@ type DatasetSpec struct {
 	// Conf map[string]string `json:"conf,omitempty"`
 	Local  map[string]string `json:"local,omitempty"`
 	Remote map[string]string `json:"remote,omitempty"`
+	Provision bool              `json:"provision,omitempty"`
```

Comment:
this is definitely the correct way to go, but since we are working with everything under the Local map[string]string and until we remove the Local/Remote separation (which doesn't make sense) can we for the time being keep the provision flag under the Local part?